src/mayur/BankingSystem.java

```diff
@@ -0,0 +1,66 @@
+package mayur;
+
+public class BankingSystem {
+	
+		 int debitCount = 0;
+		 int creditCount = 0;
+		 int balanceCount = 0;
+		 static int totalDebitCount = 0;
+		 static int totalCreditCount = 0;
+		 static int totalBalanceCount = 0;
+		 
+
+		 void creditAmt(int creditAmount) {
+			 balanceCount = balanceCount+creditAmount;
+			 System.out.println(creditAmount + " Rs. credited to your account");
+			 creditCount++;
+			 totalCreditCount++;
+		 }
+
+		 void debitAmt(int debitAmount) {
+
+			 balanceCount = balanceCount-debitAmount;
```

Comment:
You should check if (totalBalance >= requestedBalance ) then only you should debit amt from main balance